tests/cypress/integration/components/button.js

```diff
@@ -0,0 +1,22 @@
+describe('Button Component tests', function () {
```

Comment:
Can we check the consistency of the each of the components describes

Capitalisation of random letters and conventions